src/public/index.ejs

```diff
@@ -309,6 +312,50 @@ fn main() {
     println!("{:?}",response_url);
 }
 </code>
+</pre>
+                        </div>
+                        <div class="tab-pane fade" id="php-tab-pane" role="tabpanel" aria-labelledby="php-tab" tabindex="0">
+<pre><code class="language-php">
+// Upload file
+$file_path = 'file.jpg';
+$curl = curl_init();
+$curl_options = [
+	CURLOPT_CONNECTTIMEOUT => 60,
+	CURLOPT_CUSTOMREQUEST => 'PUT',
+	CURLOPT_RETURNTRANSFER => 1,
+	CURLOPT_URL => 'https://waifuvault.moe/rest',
```

Comment:
use `<%- process.env.BASE_URL; -%>/rest`

```php
CURLOPT_URL => '<%- process.env.BASE_URL; -%>/rest',
```

Reply:
that hurts my brain